src/video.js

```diff
@@ -64,3 +66,47 @@ export const checkVideoBidSetup = hook('sync', function(bid, adUnit, videoMediaT
 
   return true;
 }, 'checkVideoBidSetup');
+
+export function registerVASTTrackers(tracker) {
+  vastTrackers.push(tracker);
+}
+
+export function insertVastTrackers(trackers, vastXml) {
```

Comment:
this logic already exists in the Video Module Library. Why not reuse existing code ? 
See `getVastXmlWithTracking` https://github.com/prebid/Prebid.js/blob/master/libraries/video/shared/vastXmlEditor.js#L8

Reply:
I wasn't aware of this library. I would say not reuse this existing code avoids to run `querySelectorAll` and loop on it for each tracker to insert.